common/flatpak-transaction.c

```diff
@@ -289,6 +290,21 @@ flatpak_transaction_progress_get_bytes_transferred (FlatpakTransactionProgress *
   return self->total_transferred;
 }
 
+/**
+ * flatpak_transaction_progress_get_start_time:
+ * @self: a #FlatpakTransactionProgress
+ *
+ * Gets the time at which this operation has started, as monotonic time.
+ *
+ * Returns: the start time
+ * Since: 1.1.2
+ */
```

Comment:
Maybe this is a bit silly though... the cli transaction could could just stash the start time in new_operation. Opinions?